indexer/data_header.hpp

```diff
@@ -71,6 +72,7 @@ namespace feature
     /// @name Serialization
     //@{
     void Save(FileWriter & w) const;
+    void SaveRegionData(FileWriter & w) const;
```

Comment:
Region info section may be absent, it is not mandatory. I don't feel this change mandates incrementing data format version. The reason for a separate section is that the data is subclassed from `MetadataBase`, and such structures have been granted separate sections until now. Also, this region data section can grow quite big.